### ISSUE TITLE
Function that doesn't use a passed variable (small issue)

### ISSUE THREAD

**Issue by DFreireF** [xaratustrah/iq_suite#24]
At [this code line](https://github.com/xaratustrah/iqtools/blob/df380b3051ba59693e97a5f849bd5f389569de87/iqtools/iqbase.py#L258) you are passing the xx variable (2D array of the frequency in function of time and power xx[time, power] = f(time, power)) which is not necessary for the frame sum (and you dont use it) since zz = power (time, frequency), so you have already there the "frequency frames". 

In summary, please remove the passed variable xx for calling the function.

**Comment by xaratustrah** [comment]
solved in v3.3.4
